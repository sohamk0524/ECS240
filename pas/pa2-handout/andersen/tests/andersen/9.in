c Test case from the lecture slide 07
p 4 6
s 0 1 -1 3
s 0 3 -1 2
s 0 2 -1 1
s 0 2 0 1
s 1 2 0 3
s 0 4 1 1
