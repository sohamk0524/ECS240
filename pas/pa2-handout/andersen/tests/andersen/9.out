pt 1 2
pt 1 3
pt 2 1
pt 2 2
pt 2 3
pt 3 2
pt 4 1
pt 4 2
pt 4 3
